src/appMain/hmi_capabilities.json

```diff
@@ -462,8 +466,252 @@
                         "rdsDataAvailable": true,
                         "signalChangeThresholdAvailable": true,
                         "signalStrengthAvailable": true,
+                        "sisDataAvailable":true,
                         "stateAvailable": true
                     }
+                ],
+                "audioControlCapabilities": [
+                {
+                        "moduleName": "audio",
+                        "sourceAvailable": true,
+                        "volumeAvailable": true,
+                        "equalizerAvailable": true,
+                        "equalizerMaxChannelId": 80
```

Comment:
why sdl_hmi supports up to 100 equalizer Channels, but here the maxim id is 80? It shall be consistent across multiple repositories. For the demo and testing purposes, a small number ( for example, 3 to 10) is enough.